src/poprox_recommender/paths.py

```diff
@@ -0,0 +1,5 @@
+from pathlib import Path
+
+
+def project_root() -> Path:
+    return Path(__file__).parent.parent.parent
```

Comment:
This little utility function helps us find the root path of the repo, which makes it easier to find the `models` folder we have set up with DVC as the destination for downloaded model checkpoints/artifacts